fuzz.py

```diff
@@ -33,6 +33,7 @@
         line_length=st.just(88) | st.integers(0, 200),
         string_normalization=st.booleans(),
```

Comment:
```suggestion
        string_normalization=st.booleans(),
        preview=st.booleans(),
```
While we're here let's add this too.